fe/fe-core/src/main/java/org/apache/doris/catalog/Tablet.java

```diff
@@ -246,6 +247,34 @@ public Multimap<Long, Long> getNormalReplicaBackendPathMap() throws UserExceptio
         return map;
     }
 
+    // for cloud mode without ConnectContext. use BE IP to find replica
+    protected Multimap<Long, Long> getNormalReplicaBackendPathMap(String be_address) throws UserException {
```

Comment:
Should this function be implemented in cloud tablet?

Reply:
yes, it did. there's call from CloudTablet to this